libatomic_ops.yaml

```diff
@@ -0,0 +1,10 @@
+name: libatomic_ops
+config-opts:
+  - --enable-shared
+  - --disable-silent-rules
+sources:
+  - type: archive
+    url: https://github.com/ivmai/libatomic_ops/archive/v7.6.8.tar.gz
+    sha256: e133f6ffe0a42a69fcd73bf0bad1423fdbfb39ffad3a0fd7395a31ec1b7be94c
+post-install:
+  - install -p -D -m 0644 "COPYING" -t "${FLATPAK_DEST}/share/licenses/libatomic_ops/";
```

Comment:
The files isn't exported, it's part of Sources extension.

Reply:
Done.

https://github.com/flathub/flathub/pull/801/commits/de93f9e911a070beb1913aade48a4b7f544a2eb9#diff-98f69bafbf5ace62e811e5d696f17812L10